docs/design_decisions.md

```diff
@@ -97,7 +119,18 @@ Een developer moet bij het coderen begrijpen wat de code betekent, om fouten in
 
 _**Kanttekening**_
 
-De lengte van de enumeratiewaarden zal zoveel mogelijk beperkt moeten worden. 
+De lengte van de enumeratiewaarden zal zoveel mogelijk beperkt moeten worden (Zie DD2.6). 
+
+### DD2.4 Gebruik zo mogelijk boolean i.p.v. een enumeration
+Eigenschappen die functioneel alleen een waarde Ja/aan/waar of Nee/uit/onwaar kunnen hebben, worden gedefinieerd als boolean. We gebruiken dus geen enumeratie zoals [J,N] voor dit soort situaties.
+
+### DD2.5 Gebruik zo mogelijk string i.p.v. een enumeration
+Wanneer een gegeven in het informatiemodel gedefinieerd is als een enumeratie, maar de enumeratiewaarde wordt door gebruikers van de API alleen gebruikt wordt om als tekst te tonen aan eindgebruikers (mensen), definiëer het gegeven dan als string (niet als enumeratie) in de API.
```

Comment:
er zitten wat taalfouten in deze zin (heb ik zeker zelf ingetypt ;) )

Reply:
Is gecorrigeerd.